src/routes/profileRouter.js

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import * as profileControl from '../controllers/profileController';
+import isLoggedIn from '../middlewares/authenticate';
+
+const profileRouter = express.Router();
+
+profileRouter.put('/user/profile/:userId',isLoggedIn ,(req, res, next) => {
+  const userId = req.params.userId;
+  try {
+    profileControl.default.updateProfile(userId, req.body, res, next);
+  } catch (error) {
+    res.status(500).json({
+      'Error Message:': 'An Error Has Occured, Try Again!',
+      Error: error
+    });
+    next(error);
+  }
+});
+
+profileRouter.get('/user/profile/:userId', (req, res, next) => {
```

Comment:
This endpoint is not documented, add its swagger documentatio

Reply:
Documented.